frontend/tests/common/utils.ts

```diff
@@ -0,0 +1,40 @@
+import { ROUTES } from "src/common/constants/routes";
+import { TEST_URL } from "./constants";
+import { expect, Page } from "@playwright/test";
+import { getTestID } from "tests/utils/selectors";
+
+export function goToWMG(page: Page) {
+  return Promise.all([
+    page.waitForResponse(
+      (resp: { url: () => string | string[]; status: () => number }) =>
+        resp.url().includes("/wmg/v1/filters") && resp.status() === 200
+    ),
+    page.goto(`${TEST_URL}${ROUTES.WHERE_IS_MY_GENE}`),
+  ]);
+}
+
+export const selectAndDeselectOption = async (
+  page: Page,
+  filterName: string
+) => {
+  const FIRST_OPTION = '[data-option-index="0"]';
+  // click the filter
+  await page.getByTestId(filterName).click({ position: { x: 0, y: 0 } });
```

Comment:
can we add a comment in code to explain the need to specifically click on x 0 and y 0 vs. the default Playwright behavior? Thank you!

Reply:
It has been resolved now